### DISCUSSION TITLE
Could RavenDb be used to in a Twitter-like clone, as the backing DB?

### DISCUSSION THREAD

**Discussion by PureKrome** [ravendb/ravendb#15234] [Q&A] [answered]
👋🏻 Asking for about a bazillion friends who are a bit over the Elon-Twitter-💩

Could v6+ of RavenDb be used for a twitter-like clone? I thought one of the main concepts behind Twitter was the Graph Theory. And this has been an experimental item in version 4 and 5. 

Or have I misunderstood this (which is what I'm guessing, here)?

**Comment by georgiosd** [comment]
Now this is a topic that interests me!

Why v6+? v5 seems pretty good to me. AFAIK the graph queries are not coming back but can you define what exactly twitter seems to do as "graph"?

**Reply by PureKrome** [reply]
> Why v6+?
and
> AFAIK the graph queries are not coming back

https://ravendb.net/articles/request-for-comments-removing-graph-queries-from-ravendb

so yeah, that's why.

>  but can you define what exactly twitter seems to do as "graph"?

not sure?

**Reply by georgiosd** [reply]
Maybe it's just me, but I haven't noticed twitter doing anything that advanced with graph that you can't do without a graph database. 

If you disagree, I'd love to be proven wrong (with a specific example) :)

**Comment by ayende** [comment, marked as answer]
The short answer is that very likely yes.
You generally won't need to do graph queries for Twitter, their FlockDB was just neighbors, IIRC.

**Comment by ayende** [comment]
You might also want to watch this

https://youtu.be/z_EdHkDoxoE
